src/riscv_private.h

```diff
@@ -227,6 +236,18 @@ struct riscv_internal {
      */
     uint32_t last_csr_sepc;
 #endif
+
+#if RV32_HAS(EXT_V)
+    vreg_t V[N_RV_REGS];
+
+    uint32_t csr_vstart; /* Vector start position */
```

Comment:
This is not very important either. Not sure whether this project will support RV64. 
 If supporting 64 bits architecture is on TODO list, then maybe we need to mention about the register width here

@jserv do you have a plan about this?

Address | Privilege | Name | Width (bits) | Description
-- | -- | -- | -- | --
0x008 | URW | vstart | XLEN | Vector start position
0x009 | URW | vxsat | 32 | Fixed-Point Saturate Flag
0x00A | URW | vxrm | 32 | Fixed-Point Rounding Mode
0x00F | URW | vcsr | XLEN | Vector control and status register
0xC20 | URO | vl | XLEN | Vector length
0xC21 | URO | vtype | XLEN | Vector data type register
0xC22 | URO | vlenb | XLEN | VLEN/8 (vector register length in bytes)

Reply:
> This is not very important either. Not sure whether this project will support RV64. If supporting 64 bits architecture is on TODO list, then maybe we need to mention about the register width here, after all not all the register is `XLEN` wide
> @jserv do you have a plan about this?

Yes, I do expect to support RV64 once the system emulation for RV32 is consolidated.

Reply:
Then I suggest to add comment here, otherwise the contributor may not notice this one in the future